### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by BenMukebo**
> ## Here are what was done to implement a popup window that includes the project details in both desktop and mobile:
> 
> - [ ] Implement the following interactions:
>  - [ ] Appears the popup with details about the project when  user clicks (or taps) the button to check project details 
>  - [ ] Disappears the popup when the user clicks (or taps) the close (X) button,  
> - [ ] Use a JavaScript array to store all of the information for all projects. 
> - [ ] Implement popups for both mobile and desktop screen sizes:
> - [ ]  Fix all linter errors,
> - [ ] Update the README file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -145,9 +146,39 @@ <h3>Multi-Post Stories Gain+Glory</h3>
...
+            <li>JavScript</li> -->
+      <!-- <li>html</li> -->
+      <!-- </ul>
+          <p>Project_1 Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been
+            the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and
+            scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry.
+            Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a
+            galley of type and scrambled it 1960s with the relea</p>
+          <div class="btns-popup d-flex flex-center">
+            <button type="button" class="btn-card d-flex flex-center">See Live <img
```
Comment (dasileker) [inline index.html]: - [x] Please follow the **Dry rule** for the HTML, do not leave any commented codes in your project.

File: js/work-section.js [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+      'html',
+    ],
+    technologies: [
+      'Codekit',
+      'GitHub',
+      'Javascript',
+      'Bootstrap',
+      'Terminal',
+      'Codepen',
+    ],
```
Comment (dasileker) [inline js/work-section.js:29]: - [ ] I do not see this object in any place when I did opened the First card on mobile view see the screenshot 👇

onm

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @BenMukebo,** 

### **To highlight 🔊**

- The PR is nice.
- The commit are good.
- Linters are passing.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/work-section.js (hunk 2 above)
Comment (BenMukebo) [reply js/work-section.js:29]: It seems that The changes you have requested are supposed to be implemented in the mobile version and I have already implemented them,
In the mobile version, we only have three languages that are displayed and it's only in the desktop version where we have all the technologies above in the object displayed as in the Figma design and I have already implement all these, can you please check out this once again?

File: index.html (hunk 1 above)
Comment (BenMukebo) [reply index.html]: thanks

**Review by barackm — APPROVED** [review]
Hi @BenMukebo, 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**